hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/datanodes/datanodes.tsx

```diff
@@ -420,6 +423,25 @@ export class Datanodes extends React.Component<Record<string, object>, IDatanode
       showDataFetchError(error.toString());
     });
   };
+  
+  removeDatanode= async (selectedRowKeys: any) => {
+    try {
+      await axios.put('/api/v1/datanodes/remove', selectedRowKeys);
+      //Load Datanodes after removal
+      this._loadData();
+      this.setState({
```

Comment:
Since
```
this.setState({
        loading: false,
        selectedRowKeys: []
});
```
is being called in both the catch and try statement as the final state change, can we instead shift it to a `finally` block?